hack/packages/package-tools/cmd/vendir-sync.go

```diff
@@ -25,7 +25,6 @@ var packageVendirSyncCmd = &cobra.Command{
 
 func init() {
 	rootCmd.AddCommand(packageVendirSyncCmd)
-	packageVendirSyncCmd.Flags().StringVar(&packageRepository, "repository", "", "Package repository of the packages to be synced")
```

Comment:
same as above, i would like to keep it optional

Reply:
This one is slightly more difficult than `package-bundle-generate` and `push-package-bundle`. Currently, the way vendir-sync was run loop through files in a particular repo directory, if repository was specified with `PACKAGE_REPOSITORY`. Here is the code:

```go
for _, file := range files {
		if file.IsDir() {
			fmt.Printf("Syncing package %s\n", file.Name())
			packagePath := filepath.Join(repositoryPath, file.Name())
			if err := syncPackage(packagePath, toolsBinDir); err != nil {
				return err
			}
		}
	}
```

Now that the repository directories are gone, I'll need to figure out which packages are in the requested repository. 

What I plan to do is load the `packages/package-values.yaml file`, collect the package names in the provided repository, then do vendir sync for those packages.

Reply:
Done